.env

```diff
@@ -0,0 +1,15 @@
+# Important: NEVER ADD SECRETS OR CREDENTIALS TO THIS FILE
+# Any consistent secrets required in development should be added to config/credentials.yml.enc
+# Overrides that you do not want to check into git should be added to .env.local and/or .env.test.local
+
+# CT_BYPASS_AUTH is used to control whether or not HSES login is used
+export CT_BYPASS_AUTH=false
+
+# CT_CURRENT_USER_UID sets the HSES user id when bypassing authentication
+export CT_CURRENT_USER_UID=fake.testuser@test.com
+
+# CT_CURRENT_USER_NAME sets the user's real name when bypassing authentication
+export CT_CURRENT_USER_NAME="Fake Test-User"
+
+# CT_USE_REAL_API_DATA forces ApiDelegator to connect to real API endpoints
+export CT_USE_REAL_API_DATA=false
```

Comment:
Definitely looking for feedback about the decision to just put this in `.env` instead of `sample.env`

My thinking was that doing it this way will help with issues I've seen in the past where people add settings to the gitignore'd `.env` and forget to update `sample.env`

Reply:
[thought] hm, since we're not putting secrets in here or i guess it's fine? in my experience `.env` files are usually hidden since they have secrets and that's the purpose of the sample.env file.

Reply:
[nitpick] technically speaking we don't need to add it here, as false/nil will be handled similiarly, but since we don't have a sample.env it's probably good for awareness that we have it. (especially if we're not documenting ALL env variables in the README)

Reply:
Yeah, this file is doing double duty as setting default values and also documenting everything that can be set.